src/vs/vscode.proposed.d.ts

```diff
@@ -1127,4 +1127,13 @@ declare module 'vscode' {
 
 	}
 	//#endregion
+
+	//#region Tasks
+	export interface TaskPresentationOptions {
+		/**
+		 * Controls whether the task is executed in a specific terminal group using split panes.
+		 */
+		terminalGroup?: string;
```

Comment:
I'm leaning more towards calling this `group`. That way we aren't tied to any specific presenter.

Reply:
@alexr00 I don't think you can call it `group`. There's already a `group` property for tasks. It's used to specify if the task is the default build task or default test task. I'm assuming they would conflict? See [this](https://code.visualstudio.com/docs/editor/tasks) link for details, 

```
{
    // See https://go.microsoft.com/fwlink/?LinkId=733558
    // for the documentation about the tasks.json format
    "version": "2.0.0",
    "tasks": [
        {
            "type": "typescript",
            "tsconfig": "tsconfig.json",
            "problemMatcher": [
                "$tsc"
            ],
            "group": {
                "kind": "build",
                "isDefault": true
            }
        }
    ]
}
```

Reply:
Nevermind...I skimmed the PR and hadn't realized the "group" property was moved into presentation. Ignore me. :D

Reply:
I have now renamed `terminalGroup` to just `group`.